ui/perfherder/helpers.js

```diff
@@ -0,0 +1,334 @@
+import chunk from 'lodash/chunk';
+
+import { tValueCareMin, tValueConfidence, phTimeRanges } from './constants';
+import { getApiUrl, createQueryParams } from '../helpers/url';
+// TODO move getData into a the global helpers file if using for everything
+import { getData } from '../intermittent-failures/helpers';
```

Comment:
This is likely pulling in `moment` too, since it's defined in that helpers file.
We should probably move `getData()` to/combine it with the functions in `js/helpers/http.js` (we'll probably need a helper for fetching for bug 1473777 anyway).